src/query/settings/src/settings_getter_setter.rs

```diff
@@ -339,8 +339,8 @@ impl Settings {
         Ok(self.try_get_u64("enable_table_lock")? != 0)
     }
 
-    pub fn get_enable_stage_udf_priv_check(&self) -> Result<bool> {
-        Ok(self.try_get_u64("experiment_enable_stage_udf_priv_check")? != 0)
+    pub fn get_enable_new_rbac_check(&self) -> Result<bool> {
```

Comment:
get_enable_new_rbac_check  -> get_enable_experimental_rbac ?

"new" is easy to become a outdated naming, because "new" is always relevant to others, sometimes something even newer may exists.

Reply:
Done